popup.js

```diff
@@ -12,12 +12,31 @@ function sendRequest(data) {
 }
 
 jQuery(document).ready(function($) {
-    //send message request to content script
-    $('#btnUpload').click(function() {
-        sendRequest({
-            type: 'addJson',
-            jsontext: $('#jsontext').val()
-        });
+    $('#form-import').submit(function(e) {
```

Comment:
@J2TeaM This is my silly mistake :((, it was copy from bootstrap sample,but actually i also needn't form tag, and do you think `form submit` better than single click event.